autoload/ale/fixers/format.vim

```diff
@@ -0,0 +1,25 @@
+" Author: soywod <clement.douin@gmail.com>
+" Description: Integration of Elm with ALE.
+
+call ale#Set('elm_format_executable', 'elm-format')
+call ale#Set('elm_format_use_global', 1)
+call ale#Set('elm_format_options', '--yes')
+
+function! ale#fixers#format#GetExecutable(buffer) abort
+    return ale#node#FindExecutable(a:buffer, 'elm_format', [
+    \   'node_modules/.bin/elm-format',
+    \   'node_modules/elm-format/bin/elm-format',
+    \   'node_modules/elm-format/index.js',
```

Comment:
For at least this file, you will have to execute it with `node` on Windows, or it could lead to the `.js` file being opened in a text editor or similar. Look at the code for the `eslint` fixer for an example.

Reply:
I made a mistake here, I just had a look on this file and it's like an installer, so I will remove it. Like `elm-make`, the `'node_modules/.bin/elm-format'` is enough

Reply:
Done